README.md

```diff
@@ -71,38 +71,90 @@ Here's [example.jsonnet](example.jsonnet):
 local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
 local sts = k.apps.v1.statefulSet;
 local deployment = k.apps.v1.deployment;
-
-local kt =
-  (import 'kube-thanos/kube-thanos-querier.libsonnet') +
-  (import 'kube-thanos/kube-thanos-store.libsonnet') +
-  // (import 'kube-thanos/kube-thanos-pvc.libsonnet') + // Uncomment this line to enable PVCs
-  // (import 'kube-thanos/kube-thanos-receive.libsonnet') +
-  // (import 'kube-thanos/kube-thanos-sidecar.libsonnet') +
-  // (import 'kube-thanos/kube-thanos-servicemonitors.libsonnet') +
-  {
-    thanos+:: {
-      // This is just an example image, set what you need
-      image:: 'quay.io/thanos/thanos:v0.9.0',
-      objectStorageConfig+:: {
-        name: 'thanos-objectstorage',
-        key: 'thanos.yaml',
-      },
-
-      querier+: {
-        replicas:: 3,
-      },
-      store+: {
-        replicas:: 1,
+local t = (import 'kube-thanos/thanos.libsonnet');
+
+local commonConfig = {
+  config+:: {
+    namespace: 'thanos',
+    version: '0.10.1',
+    image: 'quay.io/thanos/thanos:v0.10.1',
```

Comment:
At first I thought we should entirely get rid of `version`. But now I see that we use the version as part of the commonLabels.
Let's at least do 
```jsonnet
version: 'v0.10.1',
image: 'quay.io/thanos/thanos:%s' % self.version,
```

If the version can be extract as easily as splitting the image at `:` and using the letter part, we should try that. Using both `version` and `image` feels too duplicate.

Reply:
works for me

Reply:
done